server/Schema/feeds.js

```diff
@@ -0,0 +1,38 @@
+import mongoose from "mongoose";
+
+const FeedSchema = new mongoose.Schema({
+  userName: {
+    type: String,
+    required: true,
+  },
+  dp: {
+    type: String,
+    // required: true,
```

Comment:
Better remove the commented code before pushing.

Reply:
Removed commented codes